library/Core/src/Juvix/Core/Application.hs

```diff
@@ -103,22 +127,12 @@ instance Bitraversable (Arg' ext) where
   bitraverse _ _ (VarArg x) = pure $ VarArg x
   bitraverse f g (TermArg t) = TermArg <$> bitraverse f g t
 
-type Arg = Arg' IR.NoExt
-
-argToTake :: Alternative f => Arg' ext ty term -> f (Take ty term)
-argToTake (TermArg t) = pure t
-argToTake _ = empty
-
-argToReturn :: Alternative f => Arg' ext ty term -> f (Return' ext' ty term)
-argToReturn = fmap takeToReturn . argToTake
-
--- |
--- An argument to a partially applied primitive, which must be
--- fully-applied itself.
+-- | An argument to a partially applied primitive, which must be fully-applied
+-- itself.
 data Take ty term = Take
-  { usage :: Usage.T,
-    type' :: ty,
-    term :: term
+  { usage :: Usage.T, -- < Usage annotation for quantified types.
+    type' :: ty, -- < The type of the term.
+    term :: term -- < The term itself.
```

Comment:
```suggestion
  { usage :: Usage.T, -- ^ Usage annotation for quantified types.
    type' :: ty, -- ^ The type of the term.
    term :: term -- ^ The term itself.
```

Reply:
I think they are actually both incorrect, as the comments are located after the comma. Maybe one of these files where the formatter fails? I will have a look.

Reply:
Ormolu is not fixing it, so I will do it by hand.